readme.md

```diff
@@ -47,6 +47,7 @@ Please raise a [Pull Request](https://github.com/wizztjh/awesome-webvr/pulls) to
 
 ### Demos
 
+- [ArtSocks](https://art-socks.herokuapp.com/demo.html) - An experiment with VR and web sockets by [dziemid](https://twitter.com/dziemid) (Google Cardboard)
```

Comment:
sure - updated README with more details - let me know if that is what you are looking for.

In more detail: it is proof of concept for virtual gallery using ThreeJS in VR - "features" -  display art work, changing between art works. Using virtual pointer to share where the user is looking and web sockets to sync multiple clients.